examples/sw/simple_system/pa_test/pa_test.c

```diff
@@ -0,0 +1,118 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "simple_system_common.h"
+
+#define PMP_NAPOT 0x18
+#define PMP_R     0x01
+#define PMP_W     0x02
+#define PMP_X     0x04
+
+#define MSTATUS_MPP_M (3 << 11)
+#define MSTATUS_MPP_U (0 << 11)
+
+
+typedef struct  {
+  uintptr_t pointer0;
+  uintptr_t pointer1;
+} fat_ptr_t;
```

Comment:
Although this is indeed a fat pointer, I would suggest instead a more specific name. Maybe something along the lines of `auth_ptr_t`, `signed_ptr_t`, etc.